sdk/digitaltwins/digital-twins-core/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-preview.2 (Unreleased)
+## 1.0.0-preview.2 (2020-10-22)
```

Comment:
can we get some notes? Perhaps mentioning the package rename and any model changes? You can also link to PRs for details

Reply:
Added